src/components/PokeCard/PokeCard.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './PokeCard.css';
+
+export default function PokeCard({ pokemon, ...poke }) {
+  return (
+    <div className="poke-div">
+      <h2>{pokemon.toUpperCase()}</h2>
+      <img className="img" src={`${poke.url_image}`} />
+      <p>Ability 1: {poke.ability_1}</p>
+      <p>Ability 2: {poke.ability_2}</p>
+      <p>HP: {poke.hp}</p>
+      <p>Egg Group 1:{poke.egg_group_1}</p>
+      <p>Defense:{poke.defense}</p>
+      <p>Shape: {poke.shape}</p>
+    </div>
```

Comment:
Since we are filtering by type, it might be good UX to have type_1/type_2 displaying on the page as well :)